### DISCUSSION TITLE
Emails not Received

### DISCUSSION THREAD

**Discussion by RohanAsad** [emailjs-com/help#60] [Q&A]
I am sending emails using Gmail service, and I receive a 200 response, but I am not receiving the email at the recipient address.

**Comment by xr0master** [comment]
Hi,
Try checking the spam folder, maybe your emails end up there. I also recommend checking your recipient address so that it does not have typos.

**Reply by thelovesmith** [reply]
Hello,

Is there any way to prevent emails from Emailjs from being sent to the spam folder? Especially with Outlook, clients need to get the emails in their inboxes, and they are going right to Spam.

**Reply by xr0master** [reply]
To do this, you have to check your emails in the spam analyzer.
For example https://www.mail-tester.com/

**Comment by Mhdnoor06** [comment]
I am also facing same issue but there is no mail on spam too but I can see the emails which I am sending on dashboard email history.

**Reply by xr0master** [reply]
In 75% of cases, people look at the wrong mailbox, not the recipient's address. In 20% of cases, it is sent to spam, and 5% are other problems.
If you are using a custom domain, you most likely do not have DNS records configured.

**Reply by Mhdnoor06** [reply]
I am not using any custom domain i have check the configuration with my gmail i only get test email "This test email was sent to you from the EmailJS dashboard. If you received this email, then your service has been configured successfully." leaving this nothing i can see on my gmail.
